src/store/selectors/tokens.js

```diff
@@ -1,31 +1,62 @@
 import { createSelector } from 'reselect';
+import widgets from './widgets';
+import reselect from '../../utils/reselect';
 
 const state = s => s.get('tokens');
 
 const defaultTokenPair = createSelector(
   state,
-  s => s.get('defaultTokenPair')
+  s => s.get('defaultTokenPair'),
```

Comment:
Kill me I have no bloody idea what those `s` and `p` variables are. Can you give more meaningful names please/

Reply:
yeah, i will rename them to store.

Reply:
done